src/cacao.ts

```diff
@@ -78,6 +82,45 @@ export namespace Cacao {
 
     return cacao
   }
+
+  export function verify(cacao: Cacao, options: VerifyOptions = {}) {
+    if (cacao.h.t === 'eip4361-eip191') {
+      return verifyEIP191Signature(cacao, options)
+    }
+    throw new Error('Unsupported CACAO signature type')
+  }
+
+  export function verifyEIP191Signature(cacao: Cacao, options: VerifyOptions) {
+    if (!cacao.s) {
+      throw new Error(`CACAO does not have a signature`)
+    }
+
+    let atTime = Date.now()
```

Comment:
```suggestion
    const atTime = options.atTime ? options.atTime * 1000 : Date.now()
```
If we only support `number` this would be simpler and we can remove the conditional below.